serverless/sql-databases/concepts.mdx

```diff
@@ -23,7 +23,80 @@ categories:
 </Concept>
 
 <Concept opened>
- ## Credentials
+## Backup
 
- Credentials let you connect to your database via a secret key.
-</Concept>
\ No newline at end of file
+A database backup is a copy of the operational state, architecture and stored data of a database. It allows to restore a database if it crashes, is corrupted or is lost.
+</Concept>
+
+<Concept opened>
+## Cold start
+
+Cold start is the time an idle database takes to handle the first request. It usually takes less than five seconds.
+</Concept>
+
+<Concept>
+## Connection pooling
+
+Connection pooling is a cache of database connections maintained so that the connections can be reused when future requests to the database are required. It enhances performance of executing commands on a database. 
+</Concept>
+
+<Concept>
+## Credentials
+
+Credentials let you connect to your database via a secret key.
+</Concept>
+
+<Concept>
+## Engine
+
+A database engine is the software that stores and retrieves your data from a database. Serverless SQL databases use the PostgreSQL engine.
+</Concept>
+
+<Concept>
+## High availability
+
+Databases are replicated on a copy of themselves that is not accessible by the user. This "standby" database guarantees that the service remains available in case of failure.
+</Concept>
+
+<Concept>
+## Idle
+
+A database switches to an idle state if it receives no query for 15 minutes. You are not billed for compute resources while your database is in an idle state.
+</Concept>
+
+<Concept>
+## Logs
+
+Logs register the activity of your database. they provide useful information for debugging or to know more about the behavior and activity of your databases.
+</Concept>
+
+<Concept>
+## PostgreSQL
+
+PostgreSQL is an open source object-relational database system known for its reliability, robustness, and performance. 
+</Concept>
+
+<Concept>
+## Region and AZ
+
+<Macro id="region-and-az" />
+
```

Comment:
To be removed for now ? (or completed with existing definitions from other products?)

Reply:
The macro parses a cross-product definition as it is widely reused content